### PR TITLE
Added forcing discovery of already discovered PDRs

### PR DESCRIPTION
**Pull request by cappelaere**
**Summary:** Summary of changes

Addresses [CUMULUS-528: Force PDR discovery] (https://bugs.earthdata.nasa.gov/browse/CUMULUS-528)

## Changes

* Added optional force attribute in Configuration to force the discovery of PDR's that have already been parsed and stored in S3
* ...

## Test Plan
Things that should succeed before merging.

- [ x] Unit tests
- [ ] Adhoc testing
- [ ] Update CHANGELOG
- [ ] Run `./bin/eslint-ratchet` and verify that eslint errors have not increased
  - [ ] Commit `.eslint-ratchet-high-water-mark` if the score has improved

Reviewers: @scisco @laurenfrederick

### CODE REVIEW

File: packages/ingest/pdr.js [hunk 1]
```diff
@@ -27,7 +27,8 @@ class Discover {
     collection,
     provider,
     useList = false,
-    folder = 'pdrs'
+    folder = 'pdrs',
+    force = false
```
Comment (scisco) [inline packages/ingest/pdr.js]: Please add tests or this change

File: tasks/discover-pdrs/index.js [hunk 2]
```diff
@@ -30,7 +30,9 @@ function discoverPdrs(event) {
       bucket,
       collection,
       provider,
-      config.useList
+      config.useList,
```
Comment (scisco) [inline tasks/discover-pdrs/index.js]: Please update the schemas for this change and add tests

File: packages/ingest/pdr.js (hunk 1 above)
Comment (cappelaere) [reply packages/ingest/pdr.js]: Done

File: tasks/discover-pdrs/index.js (hunk 2 above)
Comment (cappelaere) [reply tasks/discover-pdrs/index.js]: Done

**Comment by cappelaere** [comment]
Ready to merge

**Comment by cappelaere** [comment]
Should be good to go.

**Comment by scisco** [comment]
@cappelaere the tests are failing

**Pull request merged**